src/parser/content-parser.php

```diff
@@ -32,9 +34,11 @@ public function __construct( $block_registry = null ) {
 	 *
 	 * @return array|WP_Error
 	 */
-	public function parse( $post_content, $post_id = null ) {
+	public function parse( $post_content, $post_id = null, $blocks_to_skip = [], $blocks_to_include = [] ) {
```

Comment:
How about putting these into a combined parameter? I'm picturing something like this:

```php
public function parse( $post_content, $post_id = null, $options = [] ) {
    // ...
    $this->block_to_include = $options[ 'include' ] ?? [];
    $this->blocks_to_skip = $options[ 'exclude' ] ?? [];
```

This cleans up the method signature a bit and will make it easier to add more options without making the method signature blow up.

Reply:
Yip, agreed. Done that